data/mods/Aftershock/mobs/robots.json

```diff
@@ -237,5 +237,159 @@
     "death_drops": { "groups": [ [ "robots", 4 ], [ "manhack", 1 ], [ "turret_searchlight", 1 ] ] },
     "death_function": [ "BROKEN" ],
     "extend": { "flags": [ "HIT_AND_RUN" ] }
+  },
+  {
+    "id": "mon_utilibot",
+    "type": "MONSTER",
+    "name": "utility robot",
+    "looks_like": "mon_hazmatbot",
+    "description": "One of the many models of utility robot formerly in use by government agencies, private corporations, and civilians alike.",
+    "default_faction": "utility_bot",
+    "species": [ "ROBOT" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 120,
+    "speed": 70,
+    "material": [ "steel" ],
+    "symbol": "R",
+    "color": "green",
+    "morale": 80,
+    "aggression": 0,
+    "anger_triggers": [ "HURT" ],
+    "armor_bash": 8,
+    "armor_cut": 8,
+    "armor_fire": 4,
+    "armor_acid": 6,
+    "vision_day": 50,
+    "death_function": [ "BROKEN" ],
+    "death_drops": { "groups": [ [ "broken_robots", 3 ] ] },
+    "flags": [ "SEES", "HEARS", "ELECTRONIC", "NO_BREATHE", "PRIORITIZE_TARGETS", "PATH_AVOID_DANGER_1" ]
+  },
+  {
+    "id": "mon_craftbuddy",
+    "type": "MONSTER",
+    "name": { "str": "craftbuddy", "str_pl": "craftbuddies" },
+    "description": "A mobile crafting station used by workers in mines, on oil rigs, and in other remote locales.  In it's active state, the craftbuddy merely follows its user.  In it's deactivated state, the craft buddy is useable as a toolbench and multipurpose workstation.",
```

Comment:
I sort of dont like monsters that only work partially or require moving through hoops to make them work . I think having to constantly activate/deactivate this to make it work is very awkward.

I think it would be better to  wait on this bot until its also possible to use its workbench while its active.

Reply:
I'm good to remove this. :)